commerce_coordinator/apps/core/clients.py

```diff
@@ -89,13 +88,10 @@ def log_request_exception(self, ext_logger, exc):
                 derived class) is running in.
             response: A RequestException object from the requests library.
         """
-        ext_logger.error(exc)
-        ext_logger.info('Request URL: %s', exc.request.url)
-        ext_logger.info('Request method: %s', exc.request.method)
-        ext_logger.info('Request body: %s', exc.request.body)
-        ext_logger.debug('Request headers: %s', exc.request.headers)
-        ext_logger.info('Response status: %s %s', exc.response.status_code, exc.response.reason)
-        ext_logger.info('Response body: %s', exc.response.text)
+        ext_logger.error(
+            f"Request Exception - URL: {exc.request.url}, "
```

Comment:
Can we make this error message more descriptive.
Something like:
The API client call for url: {request_url} with request_body: {request_body} failed with status_code: {status_code}, response_body:{response.text} and error: {exc}
This is just an idea to make it more descriptive. Please append appropriate values/variables and make changes where necessary